TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -60,6 +77,11 @@ static class Parameters {
                 arity = 1,
                 description = "URI of the target cluster/domain")
         String targetUriString;
+        @Parameter(required = false,
```

Comment:
Do you expect this to be used just for development, or are their reasons someone would use it in a "real" scenario?